release_features.md

```diff
@@ -22,25 +22,25 @@ The first major release of Ignition focused on the basics of simulation. The bas
     plugins include 3D scene viewer, image viewer, topic echo, topic
     publisher, world control, and world statistics.
 
-The Acropolis collection is composed by many different Ignition libraries. The
+The Acropolis collection is composed by many different Gazebo libraries. The
 collection assures that all libraries all compatible and can be used together.
 
 | Library name       | Version       | Changelog     |
 | ------------------ |:-------------:|:-------------:|
-|   ign-cmake        |       2.x     |       [Changelog](https://github.com/gazebosim/gz-cmake/blob/ign-cmake2/Changelog.md)     |
-|   ign-common       |       3.x     |       [Changelog](https://github.com/gazebosim/gz-common/blob/ign-common3/Changelog.md)    |
-|   ign-fuel-tools   |       3.x     |       [Changelog](https://github.com/gazebosim/gz-fuel-tools/blob/ign-fuel-tools3/Changelog.md)    |
-|   ign-gazebo       |       1.x     |       [Changelog](https://github.com/gazebosim/gz-sim/blob/ign-gazebo1/Changelog.md)     |
-|   ign-gui          |       1.x     |       None       |
-|   ign-launch       |       0.x     |       None       |
-|   ign-math         |       6.x     |       [Changelog](https://github.com/gazebosim/gz-math/blob/ign-math6/Changelog.md)     |
-|   ign-msgs         |       3.x     |       [Changelog](https://github.com/gazebosim/gz-msgs/blob/ign-msgs3/Changelog.md)     |
-|   ign-physics      |       1.x     |       [Changelog](https://github.com/gazebosim/gz-physics/blob/ign-physics1/Changelog.md)      |
-|   ign-plugin       |       1.x     |       [Changelog](https://github.com/gazebosim/gz-plugin/blob/ign-plugin1/Changelog.md)     |
-|   ign-rendering    |       1.x     |       None      |
-|   ign-sensors      |       1.x     |       [Changelog](https://github.com/gazebosim/gz-sensors/blob/ign-sensors1/Changelog.md)      |
-|   ign-tools        |       0.x     |       [Changelog](https://github.com/gazebosim/gz-tools/blob/ign-tools0/Changelog.md)     |
-|   ign-transport    |       6.x     |       [Changelog](https://github.com/gazebosim/gz-transport/blob/ign-transport6/Changelog.md)      |
+|   gz-cmake        |       2.x     |       [Changelog](https://github.com/gazebosim/gz-cmake/blob/ign-cmake2/Changelog.md)     |
+|   gz-common       |       3.x     |       [Changelog](https://github.com/gazebosim/gz-common/blob/ign-common3/Changelog.md)    |
+|   gz-fuel-tools   |       3.x     |       [Changelog](https://github.com/gazebosim/gz-fuel-tools/blob/ign-fuel-tools3/Changelog.md)    |
+|   gz-gazebo       |       1.x     |       [Changelog](https://github.com/gazebosim/gz-sim/blob/ign-gazebo1/Changelog.md)     |
```

Comment:
```suggestion
|   gz-sim       |       1.x     |       [Changelog](https://github.com/gazebosim/gz-sim/blob/ign-gazebo1/Changelog.md)     |
```

Here and other places :>

Reply:
[bdd7ddd](https://github.com/gazebosim/docs/pull/267/commits/bdd7ddd9c96bb8671189bd36d3d8c13a8f2b99e7)